### PR TITLE
new Forumla for atelier

### PR DESCRIPTION
**Pull request by sithlord48**
Should work but can't test on my machine due an inability to build kf5-doctools.

### CODE REVIEW

**Comment by yurikoles** [comment]
Why do we need separate formula for same source file `atcore-1.0.0.tar.xz` that is already may be installed by `atcore`?

**Comment by sithlord48** [comment]
The AtCore 1.0.0 install is a place holder until we have a real release for atelier.

**Comment by yurikoles** [comment]
@sithlord48 I had fixed most issues and pushed this formula to atelier branch, see 4622c21a12bdfff6f9f89d9811a05134e04245ed
For now, it doesn't founds atcore, but I don't know why.

**Comment by sithlord48** [comment]
Awesome thanks I will looking to this asap.

**Comment by sithlord48** [comment]
Unfortunately I am not able to get past building kdoctools as im stuck on a perl URI:Escape missing error . I tried to install it with cpanm but no luck.

**Comment by yurikoles** [comment]
@sithlord48 it needs to be insalled with normal `cpan`, just run:
```"$(brew --repo)/Library/Taps/kde-mac/homebrew-kde/tools/all-fixes.sh"```

**Comment by yurikoles** [comment]
@sithlord48 up?

**Comment by yurikoles** [comment]
Please see 2da9b8330eac81d54d649efd21a13b7184d72b04

**Comment by sithlord48** [comment]
Unfortunately I currently have no way to test this as I have no access to
Mac os.  Last I tried even with the cpan line above I has the same issue.

**Comment by patrickelectric** [comment]
ILL TEST IT 👓

File: Formula/atelier.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  #FIX URL AFTER RELEASE.
+  url "https://download.kde.org/stable/atcore/1.0.0/atcore-1.0.0.tar.xz"
+  sha256 "ffd12455c9b8db853e455a437d6c6b601e0003c6732bbc6c2828032e004530e2"
+
+  head "git://anongit.kde.org/atelier.git"
+
+  depends_on "cmake" => :build
+  depends_on "qwt"
+  depends_on "KDE-mac/kde/kf5-extra-cmake-modules" => :build
+  depends_on "KDE-mac/kde/kf5-ki18n"
```
Comment (patrickelectric) [inline Formula/atelier.rb]: There is a trailing space here.
Also, can you put the nonbuild in alphabetic order ?

File: Formula/atelier.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  depends_on "KDE-mac/kde/kf5-extra-cmake-modules" => :build
+  depends_on "KDE-mac/kde/kf5-ki18n" 
+  depends_on "KDE-mac/kde/kf5-kconfigwidgets"
+  depends_on "KDE-mac/kde/kf5-kxmlgui"
+  depends_on "KDE-mac/kde/kf5-ktexteditor"
+  depends_on "KDE-mac/kde/kf5-knewstuff"
+  depends_on "KDE-mac/kde/atcore"
+
+  def install
+
```
Comment (patrickelectric) [inline Formula/atelier.rb]: You need to define args.
`args = std_cmake_args`

**Comment by patrickelectric** [comment]
It's working now, but you are not placing the binary where spotlight or "Application" can find.

**Comment by sithlord48** [comment]
> It's working now, but you are not placing the binary where spotlight or "Application" can find.
The binary Atelier should be deployed to <HomebrewPrefix>/usr/bin if I'm not mistaken its normal for applications and spotlight to not see homebrew apps. Perhaps we should simlink the build .app to "Applications" in a post install step ?

**Comment by yurikoles** [comment]
Closing because of lack of activity.

**Pull request closed without merging**